noodle/src/main/java/com/noodle/storage/RandomAccessFileStorage.java

```diff
@@ -237,11 +237,17 @@ private void remapIndexes() throws Exception {
           throw new RuntimeException("Data is corrupted at " + file.getFilePointer());
         }
 
-        final byte[] decryptedKey = encryption.decrypt(key);
-        index.put(new BytesWrapper(decryptedKey), pos);
+        index.put(new BytesWrapper(key), pos);
 
         file.seek(file.getFilePointer() + dataSize);
       }
     }
   }
+
+  /**
+   * Converts checked exception to runtime exception so that no mandatory try-catch is required.
+   */
+  private RuntimeException toRuntimeException(Exception e) {
+    return new RuntimeException(e);
+  }
```

Comment:
@AlexTrotsenko if this method is just a wrapper why do we need it?

Reply:
My idea was to highlight common logic in handling checked exceptions and re-throwing them as unchecked exceptions thought the lib methods as a common approach.

Later this method could be improved (if required), e.g. :
- check existing exception class and wrap in runtime only it's not runtime already
- wrap all the exceptions in the library-specific exceptions, etc.